sdk/va-report-components/documentation/docs/api/ObjectHandle.md

```diff
@@ -58,14 +58,19 @@ Returns an array of [`ReportObjectResultData`](ReportObjectResultData.md) object
 
 For user selection data, the ReportObjectResultData property `columns` will never include values for `usage`, `aggregation`, or `format`.
 
-### addEventListener(eventType: string, listener: () => void)
+### addEventListener(eventType: string, listener: (event: Object) => void)
 
 Adds an event listener to the `ObjectHandle` to call the supplied listener when the specified event occurs.
 
-Event types supported:
+#### Arguments
 
+`eventType` is a string that represents the event type to listen for. Event types supported:
```

Comment:
A small tweak for the second sentence to help with localization: These event types are supported: